Process.c

```diff
@@ -513,12 +513,16 @@ void Process_toggleTag(Process* this) {
 }
 
 bool Process_setPriority(Process* this, int priority) {
-   int old_prio = getpriority(PRIO_PROCESS, this->pid);
-   int err = setpriority(PRIO_PROCESS, this->pid, priority);
-   if (err == 0 && old_prio != getpriority(PRIO_PROCESS, this->pid)) {
-      this->nice = priority;
+   if ( Process_getuid == 0 || Process_getuid == (int) this->st_uid ) {
```

Comment:
The seteuid() option seems better, in case the user does have capabilities to change other users' processes. The rationale is that I think it's not up for the htop code to decide what should and shouldn't be allowed.

One question: what happens with euid if I explicitly run the command with `sudo htop` though? I still want htop to be able to change other users' processes that way.